### PR TITLE
[ASR pipeline] correct asr pipeline for seq2seq models

### PR DESCRIPTION
**Pull request by patrickvonplaten**
# What does this PR do?

Fixes the following daily tests:

```
FAILED tests/test_pipelines_automatic_speech_recognition.py::AutomaticSpeechRecognitionPipelineTests::test_speech_to_text_leveraged
FAILED tests/test_pipelines_automatic_speech_recognition.py::AutomaticSpeechRecognitionPipelineTests::test_torch_speech_encoder_decoder
FAILED tests/test_pipelines_automatic_speech_recognition.py::AutomaticSpeechRecognitionPipelineTests::test_xls_r_from_en
FAILED tests/test_pipelines_automatic_speech_recognition.py::AutomaticSpeechRecognitionPipelineTests::test_xls_r_to_en
```

and adds a fast test for asr pipeline for `wav2vec2-2-...` model.

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/master/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/master/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/master/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

**Comment by patrickvonplaten** [comment]
cc @Narsil - will merge since you're on holiday at the moment.

File: src/transformers/pipelines/automatic_speech_recognition.py [hunk 1]
```diff
@@ -265,10 +265,19 @@ def _forward(self, model_inputs):
...
             # the pipeline (important for `partial` in microphone)
-            input_features = model_inputs.pop("input_features")
-            attention_mask = model_inputs.pop("attention_mask")
+            if "input_features" in model_inputs:
+                inputs = model_inputs.pop("input_features")
+            elif "input_values" in model_inputs:
+                inputs = model_inputs.pop("input_values")
+            else:
+                raise ValueError(
+                    "Seq2Seq speech recognition model requires either a "
```
Comment (anton-l) [inline src/transformers/pipelines/automatic_speech_recognition.py:274]:
```suggestion
                    "The speech recognition model requires either an "
```
We're probably talking not only about seq2seq here?

**Review by anton-l — APPROVED** [review]
LGTM :+1:

**Review by patil-suraj — APPROVED** [review]
LGTM!

File: src/transformers/pipelines/automatic_speech_recognition.py (hunk 1 above)
Comment (patrickvonplaten) [reply src/transformers/pipelines/automatic_speech_recognition.py:274]: It's under the `if self.type == "seq2seq":` condition, so yeah it's just the seq2seq speech recognition model

**Pull request merged**